src/daft-logical-plan/src/optimization/optimizer.rs

```diff
@@ -32,7 +33,7 @@ impl OptimizerConfig {
 impl Default for OptimizerConfig {
     fn default() -> Self {
         // Default to a max of 5 optimizer passes for a given batch.
-        Self::new(5)
+        Self::new(20)
```

Comment:
Upping this value because we want to rely more on the fixed point rule batches to handle rule interactions

Reply:
Curious to see if this has any effects on planning times. I don't think we currently have good planning benchmarks (evidenced by the fact that I could make join reordering slow without ringing any alarms :P)